core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -118,10 +119,15 @@
 
   /**
    * Creates a new instance of {@link TieredBlockStore}.
+   *
+   * @param metaManager the block metadata manager
+   * @param lockManager the lock manager
    */
-  public TieredBlockStore() {
-    mMetaManager = BlockMetadataManager.createBlockMetadataManager();
-    mLockManager = new BlockLockManager();
```

Comment:
I would recommend this way to setup the constructors.
It is unnecessary to replace `new TieredBlockStore()` by `new TieredBlockStore(null, null)`, as the latter has slightly worse readability compared to the former.

```
  public TieredBlockStore() {
    this(BlockMetadataManager.createBlockMetadataManager(), new BlockLockManager());
 }

@VisibleForTesting
 public TieredBlockStore(BlockMetadataManager metaManager, BlockLockManager lockManager) {
  mMetaManager = metaManager;
  mLockManager = lockManager;
 }
```